docs/software/programming.md

```diff
@@ -301,3 +302,24 @@ License: [MIT](https://github.com/VSCodium/vscodium/blob/master/LICENSE)
 [^3]: [8 World-Class Software Companies That Use Python](https://realpython.com/world-class-companies-using-python/)
 
 [^4]: [TIOBE Programming Community index](https://www.tiobe.com/tiobe-index/)
+
+## Git
+
+![Git logo](../assets/images/dietpi-software-programming-git.svg){: width="200" height="200" loading="lazy"}
```

Comment:
```suggestion
![Git logo](../assets/images/dietpi-software-programming-git.svg){: width="292" height="92" loading="lazy"}
```
These attributes are meant to match the actual image size. _This is to avoid Cumulative Layout Shift, especially in combination with lazy loading, as it assures that space taken by the HTML element remains the same after the actual image file was loaded. Otherwise content on the website jumps down by the size of the image once it got loaded, which can accumulate on larger pages with many images, resulting in bad user experience._